### PR TITLE
[TIMOB-23992] iOS: Disable context menu and contextual menu in Ti.UI.WebView

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23992

### CODE REVIEW

**Comment by sgtcoolguy** [comment]
test this please

File: iphone/Classes/TiUIWebView.m [hunk 1]
```diff
@@ -775,10 +775,9 @@ - (void)webViewDidFinishLoad:(UIWebView *)webView
         }
     }
     
-    // Disable user selection and the attached callout
-    BOOL disableSelection = [TiUtils boolValue:[[self proxy] valueForKey:@"disableSelection"] def:NO];
+    // Disable the context menu when selecting text
+    BOOL disableSelection = [TiUtils boolValue:[[self proxy] valueForKey:@"disableContextMenu"] def:NO];
```
Comment (sgtcoolguy) [inline iphone/Classes/TiUIWebView.m]: might want to rename the variable to match the property: disableContextMenu - rather than disableSelection.

**Review by sgtcoolguy — CHANGES_REQUESTED** [review]

File: iphone/Classes/TiUIWebView.m (hunk 1 above)
Comment (hansemannn) [inline iphone/Classes/TiUIWebView.m]: Agree. The Travis job is running for 2 days now, hopefully it's re-triggered correctly after the commit.

**Comment by hansemannn** [comment]
Hey Chris, so I finally updated the docs in @5fbc5d9 to have the Android part documented as well (not done in #8494). We should be cool now!

**Review by sgtcoolguy — APPROVED** [review]

**Pull request merged**